packages/style/src/config/maps/themes/basic/palette/_palette.scss

```diff
@@ -21,5 +21,6 @@ $theme-basic-palette: (
   color-error-hover: $color-error-hover,
   color-error-highlight: $color-error-highlight,
   color-link_color: $color-link_color,
-  color-surface: $color-surface
+  color-surface: $color-surface,
+  backdrop-color: $color-surface
```

Comment:
Are you sure it's supposed to be here?

Reply:
I'm absolutely not sure, the dist's `index.css` looks fine though